clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -1728,6 +1730,7 @@ bool CodeGenFunction::ConstantFoldsToSimpleInteger(const Expr *Cond,
   if (!AllowLabels && CodeGenFunction::ContainsLabel(Cond))
     return false;  // Contains a label.
 
+  PGO.markStmtMaybeUsed(Cond);
```

Comment:
Is this change actually NFC? Does this impact the IR?

Reply:
No. Just for checks.

Reply:
OK gotcha